packages/usa-input-mask/src/styles/_usa-input-mask.scss

```diff
@@ -0,0 +1,28 @@
+@use "uswds-core" as *;
+
+.usa-input-mask {
+  position: relative;
+  line-height: 1;
+
+  &--content {
+    position: absolute;
+    left: 2px;
+    top: 0;
+    pointer-events: none;
+    color: color("gray-50");
+    font-family: family("sans");
+    border: 1px solid transparent;
+    i {
+      visibility: hidden;
+    }
+  }
+}
+
+usa-masked,
```

Comment:
The class styles are currently not being applied to `.usa-masked`. Is this line needed?

Reply:
changed to .usa-masked... They are both needed I believe. .usa-masked is applied to the input and .usa-input-mask--content is applied to the dynamically created shell span.